pkg/context/project.go

```diff
@@ -21,10 +23,14 @@ func ResolveProject(f *pflag.FlagSet) {
 	activeProjectFilePath := path.Join(common.ConfigPath, activeProjectFilename)
 	projectFlagValue, _ := f.GetString(common.ProjectNameFlag)
 
-	if _, err := os.Stat(activeProjectFilePath); (os.IsNotExist(err) || common.GetActiveProject() == "") && projectFlagValue == "" {
+	if _, err := os.Stat(activeProjectFilePath); os.IsNotExist(err) && projectFlagValue == "" {
 		initActiveProject()
 		fmt.Println(fmt.Sprintf("Active project was not yet set, has been set to '%v'. You can set a project "+
-			"with 'strm context project <project-name>'\n", common.GetActiveProject()))
+			"with 'strm context project <project-name>'\n", user_project.GetActiveProject()))
+	}
+
+	if user_project.GetActiveProject() == "" {
```

Comment:
based on the removed line of 24, I think it misses: ` && projectFlagValue == ""`